### PR TITLE
Update asyn.sync Docstring

### PR DESCRIPTION
**Pull request by FarukOzderim**
I tried to use this function with asyncio.get_event_loop, but was unsuccessful, see #981. After spending a bit time not able to solve the issue, @isidentical helped me to save the issue :)

Adding example to the docstring in case any people have similar problems in the future.

### CODE REVIEW

File: fsspec/asyn.py [hunk 1]
```diff
@@ -60,6 +60,9 @@ async def _runner(event, coro, result, timeout=None):
 def sync(loop, func, *args, timeout=None, **kwargs):
     """
     Make loop run coroutine until it returns. Runs in other thread
+    
+    Example usage:
+        ffspec.asyn.sync(fsspec.asyn.get_loop(), func, *args, timeout, **kwargs)
```
Comment (isidentical) [inline fsspec/asyn.py:65]:
```suggestion
        fsspec.asyn.sync(fsspec.asyn.get_loop(), func, *args, timeout, **kwargs)
 ```

**Comment by isidentical** [comment]
Thanks for the PR @FarukOzderim!

**Review by isidentical — APPROVED** [review]
Thanks @FarukOzderim!

**Comment by martindurant** [comment]
There seems to be some whitespace lint:
```diff
     """
     Make loop run coroutine until it returns. Runs in other thread
-    
+
     Example usage:
         fsspec.asyn.sync(fsspec.asyn.get_loop(), func, *args, timeout=timeout, 
```

**Pull request merged**